modules/examples/lnl.nix

```diff
@@ -21,6 +21,8 @@
   system.defaults.finder.QuitMenuItem = true;
   system.defaults.finder.FXEnableExtensionChangeWarning = false;
 
+  system.defaults.screencapture.location = "/tmp";
```

Comment:
I use this as a base for my configuration, and don't necessarily want  to set this. 😄

Reply:
Oh... haha, sorry! I'll take this out 😉